example/pom.xml

```diff
@@ -63,6 +63,11 @@
             <version>1.6.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
This is now being used in several places.  Can this dependency be pulled in to the top level POM.